### PR TITLE
tests/CI: Add RHEL 9.3 and 8.9 GA to pipeline

### PR DESCRIPTION
**Pull request by tkoscieln**
Move testing to RHEL 8.9 and 9.3 runners.

This pull request includes:

- [x] adequate testing for the new functionality or fixed issue

- [ ] adequate documentation informing people about the change such as
  - [ ] submit a PR for the [guides](https://github.com/osbuild/guides) repository if this PR changed any behavior described there: https://www.osbuild.org/guides/

### CODE REVIEW

**Review by atodorov — APPROVED** [review]
LGTM

File: test/cases/regression-old-worker-new-composer.sh [hunk 1]
```diff
@@ -13,9 +13,9 @@ ARTIFACTS="${ARTIFACTS:-/tmp/artifacts}"
 source /usr/libexec/osbuild-composer-test/set-env-variables.sh
 source /usr/libexec/tests/osbuild-composer/shared_lib.sh
 
-# Only run this on x86 and rhel8 GA
+# Only run this on x86
```
Comment (jrusz) [inline test/cases/regression-old-worker-new-composer.sh:16]: That RHEL GA part was correct, we still check for it in the condition, just the version was supposed to be both 8 and 9.

**Review by jrusz — APPROVED** [review]
Just minor nitpick, does not block merge.

**Comment by thozza** [comment]
Clicked the "rebase" button, since test pipelines were in a weird shape...

File: test/cases/regression-old-worker-new-composer.sh (hunk 1 above)
Comment (tkoscieln) [reply test/cases/regression-old-worker-new-composer.sh:16]: I think it could be clear from the error mesage

**Pull request merged**